tests/snuba/api/endpoints/test_organization_events_span_indexed.py

```diff
@@ -1839,3 +1837,6 @@ def test_tag_wildcards(self):
     @pytest.mark.xfail(reason="wip: rate not implemented yet")
     def test_spm(self):
         super().test_spm()
+
+    def test_simple_measurements(self):
+        super().test_simple_measurements()
```

Comment:
Is there a point to this?

Reply:
so i can use vim-test in my local env 😛, good catch I'll remove